ethereum/beaconchain/src/main/java/tech/pegasys/artemis/datastructures/beaconchainblocks/ProposalSignedData.java

```diff
@@ -14,7 +14,7 @@
 package tech.pegasys.artemis.datastructures.beaconchainblocks;
 
 import com.google.common.primitives.UnsignedLong;
-import tech.pegasys.artemis.ethereum.core.Hash;
+import net.consensys.cava.crypto.Hash;
```

Comment:
I see in a couple of places that Hash was replaced with Bytes32. We should probably opt for consistency on type usage.

Reply:
agreed